src/main/scala/com/vngrs/lightningtalk/typeclasses/Equals.scala

```diff
@@ -1,7 +1,7 @@
 package com.vngrs.lightningtalk.typeclasses
 
 trait Equals[T] {
-  def ===(a: T, b: T): Boolean
+  def ===(a: T, b: T): Boolean;
```

Comment:
Let's change it to this because you don't need `;` in Scala. 😎

```suggestion
  def ===(a: T, b: T): Boolean
```